### DISCUSSION TITLE
[QUESTION] How to change sidebar width?

### DISCUSSION THREAD

**Discussion by cod3rshotout** [ColorlibHQ/AdminLTE#4248] [Q&A]
Hi, is there a way to change the sidebar width? Could anyone show me how to do this?

Thanks

**Comment by REJack** [comment]
For this you need to change the variable in _variables.scss and recompile them.

**Reply by TheShoaib19** [reply]
Can you please guide on how to recompile, I'm using the "Live Sass Compiler" Extension in VS code but it's giving me this error: 
Compilation Error
Error: Can't find stylesheet to import.
   ╷
11 │ @import "~bootstrap/scss/functions";
   │         ^^^^^^^^^^^^^^^^^^^^^^^^^^^
   ╵
  d:\Laravel\project\public\assets\build\scss\parts\adminlte.plugins.scss 11:9  root stylesheet
--------------------

**Reply by Saggre** [reply]
@TheShoaib19 Either `~bootstrap/scss/functions.scss` or `~bootstrap/scss/_functions.scss` does not exist OR the alias `~` in your build environment is not configured to the correct path.

**Reply by TheShoaib19** [reply]
@Saggre True there is no "/scss/functions.scss" in my bootstrap folder and when I'm creating it manually then it gives other set of errors. and dumb question to ask but can you let me know how can I configure the "correct path"?
![image](https://github.com/ColorlibHQ/AdminLTE/assets/82964146/59200479-149e-4c19-b52e-b166ffd664df)

**Reply by Saggre** [reply]
@TheShoaib19 Here `~` should point to modules, aka `<project-dir>/node_modules` so there should be a file at path `<project-dir>/node_modules/bootstrap/scss/_functions.scss`. This file should be downloaded by the package manager as admin-lte has bootstrap as its dependency. Do you have a node_moduels folder in your project?

**Reply by TheShoaib19** [reply]
@Saggre no, there is no node_modules folder in the project.

**Reply by Saggre** [reply]
@TheShoaib19 Try installing admin-lte with npm: `npm install admin-lte`

**Reply by TheShoaib19** [reply]
@Saggre tried but then there is no scss folder now, it has less.. :( The thing is I was told to include the adminLTEs 'build', 'dist' folder in my laravel project and use it from there but it's just not working but still thank you for your help :)

**Comment by Saggre** [comment]
`$sidebar-width: <width> !default;`
